.github/workflows/pr-check.yml

```diff
@@ -25,7 +74,21 @@ jobs:
 
       - run: yarn install --prefer-offline --frozen-lockfile
 
+  Backend:
+    needs: Setup
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Can you add another step here (before the yarn test) that builds the backend? (wasn't in scope but should be done).

Reply:
Thanks, I also add that one